curriculum/challenges/english/blocks/daily-coding-challenges-javascript/68d2ba1468508398389487cf.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 68d2ba1468508398389487cf
+title: "Challenge 74: Favorite Songs"
+challengeType: 28
+dashedName: challenge-74
+---
+
+# --description--
+
+Remember iPods? The first model came out 24 years ago today, on Oct. 23, 2001.
+
+Given an array of song objects representing your iPod playlist, return an array with the titles of the two most played songs, with the most played song is first.
+
+- Each object will have a `"title"` property (string), and a `"plays"` property (integer).
+
+# --hints--
+
+`favoriteSongs([{"title": "Sync or Swim", "plays": 3}, {"title": "Byte Me", "plays": 1}, {"title": "Earbud Blues", "plays": 2} ])` should return `["Sync or Swim", "Earbud Blues"]`.
+
+```js
+assert.deepEqual(favoriteSongs([{"title": "Sync or Swim", "plays": 3}, {"title": "Byte Me", "plays": 1}, {"title": "Earbud Blues", "plays": 2} ]), ["Sync or Swim", "Earbud Blues"]);
+```
+
+`favoriteSongs([{"title": "Skip Track", "plays": 98}, {"title": "99 Downloads", "plays": 99}, {"title": "Clickwheel Love", "plays": 100} ])` should return `["Clickwheel Love", "99 Downloads"]`.
+
+```js
+assert.deepEqual(favoriteSongs([{"title": "Song A", "plays": 42}, {"title": "Song B", "plays": 99}, {"title": "Song C", "plays": 75} ]), ["Song B", "Song C"]);
```

Comment:
I hope the suggestion is actually only on line 27, GitHub was being naughty here

```suggestion
assert.deepEqual(favoriteSongs([{"title": "Skip Track", "plays": 98}, {"title": "99 Downloads", "plays": 99}, {"title": "Clickwheel Love", "plays": 100} ]), ["Clickwheel Love", "99 Downloads"]);
```